cfme/infrastructure/virtual_machines.py

```diff
@@ -14,7 +14,8 @@
 from widgetastic_patternfly import (
     Button, BootstrapSelect, BootstrapSwitch, Dropdown, Input as WInput, Tab)
 from widgetastic_manageiq import (
-    TimelinesView, Accordion, ManageIQTree, SummaryTable, ConditionalSwitchableView)
+    NonJSPaginationPane, TimelinesView, Accordion,
+    ManageIQTree, SummaryTable, ConditionalSwitchableView)
```

Comment:
Imports should be sorted alphabetically.

Reply:
Done.